WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -264,7 +259,6 @@ - (UINavigationController *)notificationsNavigationController
     _notificationsNavigationController.tabBarItem.selectedImage = notificationsTabBarImage;
     _notificationsNavigationController.tabBarItem.imageInsets = [self tabBarIconImageInsets];
     _notificationsNavigationController.restorationIdentifier = WPNotificationsNavigationRestorationID;
-    _notificationsNavigationController.tabBarItem.title = @"";
```

Comment:
Yeah so this is a bad merge as 6.4 restored the labels and fixed alignments with some of the images in the tab bar.